global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "3.1.101",
+    "version": "5.0.202",
```

Comment:
I believe you need to update build pipieline. This will break the builds as we're installing core 3.1

Reply:
+ let's remove appveyour as we're using github actions now

Reply:
I have removed global.json (expecting that was what you wanted me to do).  Build is passing now